docs/proposals/test-reports/pedestrian_tracking.md

```diff
@@ -0,0 +1,127 @@
+# Implement function of automatically algorithm performance tests for Sedna Edge-cloud Re-ID
+
```

Comment:
The title "Motivation" should be added here.

Reply:
The comment has been revised